### ISSUE TITLE
Game Manuals and changing PS1 games' bootup splash screen?

### ISSUE THREAD

**Issue by PlayerGameSK** [LiEnby/chovy-sign#18]
So I just gave this tool a shot today and gotta say that so far it's been great, though I have a question... 

Basically I would love to be able to make my PS1 game bubbles appear as if they were official, including the manuals, the "warning" screen on bootup and such, the problem is that there seems to be no option to include your own manual, nor change the default bootup splash screen (which is just a very basic #ChovyProject logo), and so I've been wondering... are these options planned to be added by any chance?

**Comment by LiEnby** [comment]
I don't really know the document.dat format. I was gonna work on adding dlc support for psp games next.. and acid snake wanted the ability to add PBOOT.PBP files or smth. Idk so if probably do those. Changing the splash pic would be super easy. Vita requires there to be *something* set as an image so I just put that one there

**Comment by TheShadowRunner** [comment]
Ok figured it out, When I used Content Manager to copy the generated game from chovy-sign over to Vita, it just completely overrote the game folder on PSPEMU that the DLCs were using, so they were erased.

Now I copied the DLC back over again with vitashell to the same folder and when I launch that game with Adrenalin, DLC is back!
When I launch same game via Bubble, DLC is absent..

We're not far I'm sure!

**Comment by LiEnby** [comment]
> Ok figured it out, When I used Content Manager to copy the generated game from chovy-sign over to Vita, it just completely overwrote the game folder on PSPEMU that the DLCs were using, so they were erased.
> 
> Now I copied the DLC back over again with vitashell to the same folder and when I launch that game with Adrenalin, DLC is back! When I launch same game via bubble, DLC is absent..

if i had to guess the DLC is probably been decrypted either by pkgj or some other tool during the install process
 (like how the pbp NPUMDIMG was decrypted to a regular .iso ..)

i think because adrenaline is more like a psp cfw, it probably patches so that it can run even decrypted DLC files, but since nopspemudrm * just * patches the npdrm, and nothing else-- this doesn't work there

**Comment by TheShadowRunner** [comment]
Hi Li, thanks for your support. That DLC I actually had from a long time ago and used back then on PSP. I just copied it over to Vita using Vitashell to the correct folder ux0:pspemu/PSP/GAME/*GameID*.
I'm pretty sure you're correct in the fact it's decrypted.
On PSP at the time, it was needed to use plugin "nploader.prx" to load this DLC, then it became obsolete and support was built-in CFW directly, 
I believe Adrenaline also has it built-in hence how it works.
I wonder if that old plugin "nploader.prx" could be ported to Vita somehow to allow DLC to work, via NoPspEmuDrm maybe?
Regarding the ability for users to set custom launch screens for PSP/PS1, what do you think ? ^^;

**Comment by PlayerGameSK** [comment]
> changing the splash pic would be super easy though .. Vita requires there to be something set as an image and nothing in the iso file that would be It so I just put the chovy project logo up haha 

Yeah, I wanted mine to look as official as possible, so I got the source code, replaced the images in the GameBuilder/Resources folder, "compiled it" (a.k.a ran through the debug mode in Visual Studio because I'm dumb and couldn't figure out how to actually compile it) and used that to create them 😅 
(sorry for the late response lol I don't go on GitHub very often)
